test/runtests.jl

```diff
@@ -846,3 +846,36 @@ Tables.columnnames(::WideTable2) = [Symbol("x", i) for i = 1:1000]
     end
 
 end
+
+@testset "ByRow" begin
+    @test Tables.ByRow(x -> x^2)(1:3) == [1, 4, 9]
+    @test Tables.ByRow((x, y) -> x*y)(1:3, 2:4) == [2, 6, 12]
+    @test Tables.ByRow((x, y, z) -> x*y*z)(1:3, 2:4, 3:5) == [6, 24, 60]
+    let i = 0
+        @test Tables.ByRow(x -> (i += 1; return i))(spzeros(5)) == 1:5
+    end
+    # check that map is not what we want
+    let i = 0
+        @test map(x -> (i += 1; return i), spzeros(5)) == [1, 1, 1, 1, 1]
+    end
```

Comment:
Do we really need to test this? We don't really care how `map` is implemented in SparseArrays.

Reply:
We do not need it, but I think it is not a problem and shows the reader of tests the difference (which is not obvious I would say).